src/steps/generic.rs

```diff
@@ -220,6 +220,16 @@ pub fn run_pipx_update(run_type: RunType) -> Result<()> {
     run_type.execute(&pipx).arg("upgrade-all").check_run()
 }
 
+pub fn run_pip3_update(run_type: RunType) -> Result<()> {
+    let pip3 = utils::require("pip3")?;
+    print_separator("pip3");
+
+    run_type
+        .execute(&pip3)
+        .args(&["install", "--upgrade", "pip"])
```

Comment:
This should use --user to not overwrite the system pip.

Reply:
I believe it will not overwrite the system pip, as this is usually not writable for the user (e.g. on macOS I get a `Defaulting to user installation because normal site-packages is not writeable`). If the user wants actually to upgrade the system version he/she can invoke `topgrade` with `sudo`, no?

Reply:
This is true in Linux, however, this is what you get in Windows when not using `--user`:

```
ERROR: Could not install packages due to an EnvironmentError: [WinError 5] Access is denied: 'c:\\pyth on39\\lib\\site-packages\\pip-20.2.3.dist-info\\entry_points.txt' Consider using the `--user` option or check the permissions.
```

So we should specify `--user` explicitly